openmc/trigger.py

```diff
@@ -96,6 +97,20 @@ def add_score(self, score):
                   'it is not a string'.format(score)
             raise ValueError(msg)
 
+        # If this is a total/flux/scattering moment, use generic moment order
+        regexp = re.compile(r'-[0-9]')
```

Comment:
I'd prefer regexs be compiled when the module is loaded, or you could just use the `re.search` function directly.